Scripts/license_headers.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+if [[ "$1" == "check" ]]; then
+  update=0
+elif [[ "$1" == "update" ]]; then
+  update=1
+else
+  echo "::error::Either 'check' or 'update' must be provided as an argument"
+  exit 1
+fi
+
+if [[ "$2" != "" ]]; then
+  modified_files=$2
+else
+  modified_files=$(git diff HEAD --name-only | grep '.swift$')
+fi
+
+BASEDIR=$(dirname $0)
+errors=0
+current_year=$(date +"%Y")
+header=$(cat LICENSE_HEADER)
+cd ${BASEDIR}/..
+
+for file in $modified_files; do
+  if [[ $file == *.swift ]]; then
+    if ! grep -q Copyright "$file"; then
+      if [[ $update -eq 1 ]]; then
+        echo "$(echo -e "$header\n"; cat $file)" > $file
+        echo "Added license header in $file"
+      else
+        echo "::error::License header is missing in $file"
+        ((errors++))
+      fi
+    fi
+    if ! grep -q "$current_year" "$file"; then
+      if [[ $update -eq 1 ]]; then
+        str="s/Copyright.*Pexip/Copyright 2022-$current_year Pexip/"
```

Comment:
Is hardcoded 2022 here correct? If you add a new file in 2023, should it say Copyright 2022-2023 then?

Reply:
You're right, if the file was created in 2023 it makes more sense to say Copyright 2023; or Copyright 2023-2024 if it gets updated next year. Should be fixed now.

Reply:
Looks good now :)